modules/manage/pages/disaster-recovery/shadowing/failover-runbook.adoc

```diff
@@ -152,22 +178,76 @@ Note the replication lag to estimate potential data loss during failover. The `T
 
 A complete cluster failover is appropriate If you observe that the source cluster is no longer reachable:
 
+[tabs]
+======
+rpk::
++
+--
 [,bash]
 ----
 # Fail over all topics in the shadow link
 rpk shadow failover <shadow-link-name> --all
 ----
+--
+
+ifdef::env-cloud[]
+Cloud API::
++
+--
+[,bash]
+----
+# Get Data Plane API URL of shadow cluster
+export DATAPLANE_API_URL=`curl https://api.cloud.redpanda.com/v1/clusters/<shadow-cluster-id> \
+  -H "Content-Type: application/json" \
+  -H "Authorization: Bearer ${RP_CLOUD_TOKEN}" | jq .cluster.dataplane_api`
+
+# Fail over all topics in the shadow link
+curl -X POST "$DATAPLANE_API_URL/v1/shadowlink/<shadow-link-name>/failover" \
```

Comment:
@simon0191 Would you mind confirming that the _name_ (not ID) of the shadow link is required in both the data plane API URL (as a param) as well as the request body?

Reply:
@c-julin could you confirm?

In any case (ID or name), if you provide it in the path you don't need to provide it in the body